storybookjs/storybook
# #3377 remove old keyboard shortcuts

Issue: #3377 

## What I did

deleted them and added some notes in the migration doc

## How to test

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?
yes, added
If your answer is yes to any of these, please make sure to include it in your PR.

Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
